src/lib/_imports/components/c-card.css

```
@@ -185,4 +185,28 @@
 /* SEE: https://www.tacc.utexas.edu/education/k-12-students/high-school-camps/gencyber/ */
 :is(.card, :--c-card) h4 {
     margin-bottom: 0.375em;
-}
\ No newline at end of file
+}
+
+
+
+
+
+/* SUPPORT LIST IN CARD */
+
+/* FAQ: The Portal uses Bootstrap ".card" for ticket modal */
+:is(.card, :--c-card, [class*="card--"]) ul {
+    /* list-style: none; *//* H.P. restored bullets, M.S. does not know */
+    padding-left: 1em; /* overwrite core-styles.cms */
+}
+/* To add space between line items */
+/* FAQ: Using margin and li:not(:first-of-type) because of multi-line items */
+:is(.card, :--c-card, [class*="card--"]) li:not(:first-of-type) {
+    margin-top: 0.5em;
+}
+/* TODO: Consider changing this site wide; SEE:
+:where([role=main],main) li:not(td li) { line-height: 1.6; } */
+/* Tomas removes `:where(:not(…` (e.g. "not in portal"). He saw it needed to be
+removed elsewhere. */
+:is(.card, :--c-card, [class*="card--"]) ul:last-child {
+    margin-bottom: 2rem;
+}
```

```suggestion



/* Elements: Lists */

:--c-card ul {
    /* list-style: none; *//* H.P. restored bullets, M.S. does not know */
    padding-left: 1em; /* overwrite core-styles.cms */
}
/* To add space between line items */
/* FAQ: Using margin and li:not(:first-of-type) because of multi-line items */
:--c-card li:not(:first-of-type) {
    margin-top: 0.5em;
}
:--c-card ul:last-child {
    margin-bottom: 2rem;
}
```

- I let new styles be hierarchically within `/* Elements */`.
- I removed my comment about Bootstrap card that is less relevant than I feared.
- I removed my comment about line-height that I believe is misguided.
- I've removed a new tup-specific comment.

If I am removing anything you know should stay or be discussed, please let me know.

---

I think this looks good.